templates/core/home.html

```diff
@@ -237,26 +237,36 @@ <h3 class="font-bold mb-2 text-pyos-deep-purple">Blog Post Title {{ forloop.coun
         <h2 class="text-3xl font-bold text-center mb-12 text-pyos-deep-purple font-poppins">Recently Accepted Python Packages</h2>
         
         <div class="grid md:grid-cols-3 gap-6 mb-8">
-            <!-- Placeholder package cards -->
-            {% for i in "123" %}
+            {% for package in recent_packages %}

```

Comment:
Something is odd here. As it has neonutilities as a recent package. 

<img width="1352" height="549" alt="Screenshot 2025-09-03 at 1 47 09 PM" src="https://github.com/user-attachments/assets/28a6ef48-3fd2-4af6-bef7-e87e1b42addd" />

if you go to the website

https://www.pyopensci.org/python-packages.html

you will see a different set of newest packages. could we be sorting/ filtering using the wrong value?